crates/pop-cli/src/commands/up/parachain.rs

```diff
@@ -37,6 +39,10 @@ pub struct UpChainCommand {
 	/// Websocket endpoint of the relay chain.
 	#[arg(long)]
 	pub(crate) relay_url: Option<Url>,
+	///  Proxied account address. The proxy can act on behalf of this account.
+	/// Specify the address type, e.g. `Id(5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty)`.
```

Comment:
Assume it has to be `Id(..)` as its a `MultiAddress`? Can we also support a raw 'id' and wrap it in `Id()`, whilst still preserving the ability to use the other variants of MultiAddress?

Can a better type be used than string perhaps, as with Url?